src/calculateRentalCost.js

```diff
@@ -4,7 +4,14 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  switch (true) {
+    case days < 3:
+      return days * 40;
+    case days >= 3 && days < 7:
+      return days * 40 - 20;
+    case days >= 7:
+      return days * 40 - 50;
```

Comment:
The function name 'calculateRentalCost' is descriptive, which is good. However, the implementation uses literals directly in the expressions which makes it hard to understand what these numbers represent. Consider defining constants at the beginning of your function to give these literals a clear meaning.